e2e/_suites/fleet/stand-alone.go

```diff
@@ -51,7 +51,8 @@ func (fts *FleetTestSuite) thereIsNewDataInTheIndexFromAgent() error {
 	maxTimeout := time.Duration(common.TimeoutFactor) * time.Minute * 2
 	minimumHitsCount := 50
 
-	result, err := searchAgentData(fts.Hostname, fts.RuntimeDependenciesStartDate, minimumHitsCount, maxTimeout)
+	manifest, _ := fts.deployer.Inspect("elastic-agent")
```

Comment:
👍 I like this approach of extracting the hostname from the fleet test suite struct, and using the Inspect abstraction to get it from the provider